src/authority.rs

```diff
@@ -18,7 +18,7 @@ use std::fmt::{self, Binary, Debug, Display, Formatter};
 /// `PrefixSection` use _section_ verification: the set from which a quorum is required is all
 /// members of the section (`Section`) or of all sections matching the prefix (`PrefixSection`).
 #[derive(Serialize, Deserialize, PartialEq, PartialOrd, Eq, Ord, Clone, Copy, Hash)]
-#[allow(clippy::large_enum_variant)]
+// #[allow(clippy::large_enum_variant)]
```

Comment:
Must have dropped the commit that removed that commenting out instead of squashing it?
A good way to verify if something went wrong with history rewriting is to diff backup branch with new branch. If nothing is different then all good, if anything is different check if that was intended.

Reply:
Ah will push that again now. 2 secs